app/views/scheduled_messages/_form.html.erb

```diff
@@ -4,6 +4,13 @@
   <%= form.input :body, label: "Message body",
     hint: "Don't forget, you can format your message using #{link_to "Slack's message formatting rules",
     "https://slack.zendesk.com/hc/en-us/articles/202288908-Formatting-your-messages", target: "_blank"}.".html_safe %>
+  <%= check_box_tag :quarterly, checked: false %> Quarterly
+  <div style="display:none" class="quarterly_message_explanation">
+    This message will be sent to all employees quarterly
+    <br />
```

Comment:
any way to do this without `br` tags? I am not a frontend pro but I know they are generally not favored...